### ISSUE TITLE
Identity V3 users: need some functions

### ISSUE THREAD

**Issue by zhangzujian** [gophercloud/gophercloud#843]
Before starting a PR, please read the [style guide](https://github.com/gophercloud/gophercloud/blob/master/STYLEGUIDE.md).

[Identity API v3 - Change password for user](https://developer.openstack.org/api-ref/identity/v3/index.html#change-password-for-user)
[Identity API v3 - Add user to group](https://developer.openstack.org/api-ref/identity/v3/index.html#add-user-to-group)
[Identity API v3 - Remove user from group](https://developer.openstack.org/api-ref/identity/v3/index.html#remove-user-from-group)
[Identity API v3 - Check whether user belongs to group](https://developer.openstack.org/api-ref/identity/v3/index.html#check-whether-user-belongs-to-group)

**Comment by jtopjian** [comment]
@zhangzujian Thank you for adding some missing Identity features.

Please note that each PR must only include one feature addition, so each of the above items will need to be a separate PR.

Also note that you'll need to provide references to the Keystone python code where each feature is implemented. The API docs can't be used as a primary reference.

I have a draft contributor tutorial written here that might help out with this:

https://github.com/jtopjian/gophercloud/tree/docs-tutorial/docs/contributor-tutorial

Please let me know if you have any questions.
